apps/events/serializers/events.py

```diff
@@ -45,10 +24,9 @@ class Meta:
 class EventCreateSerializer(serializers.ModelSerializer):
     desired_participants_number = serializers.IntegerField(min_value=0, max_value=10000000, default=0, allow_null=True,
                                                            required=False)
-    location = LocationSerializer(required=True, many=False)
-    city_south_west_point = LocationSerializer(required=True, many=False)
-    city_north_east_point = LocationSerializer(required=True, many=False)
     city = serializers.CharField(max_length=50)
+    place_id = serializers.CharField(max_length=255)
+    city_location = city_serializers.CitySerializer()
```

Comment:
city_location should contain place_id, not duplicate it in both places

Reply:
Done